debezium-core/src/test/java/io/debezium/data/SourceRecordAssert.java

```diff
@@ -40,4 +40,9 @@ public SourceRecordAssert valueAfterFieldSchemaIsEqualTo(Schema expectedSchema)
         VerifyRecord.assertConnectSchemasAreEqual(null, afterFieldSchema, expectedSchema);
         return this;
     }
+
+    public SourceRecordAssert keySchemaIsEqualTo(Schema expectedSchema) {
```

Comment:
Nice, I already forgot about `SourceRecordAssert`. We should use it more often :)